testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -2128,7 +2130,7 @@ fn atlas_integration_test() {
     // Poll GET v2/attachments/<attachment-hash>
     for i in 1..10 {
         let mut attachments_did_sync = false;
-        let mut timeout = 30;
+        let mut timeout = 60;
```

Comment:
This might need to be longer -- I've had to set it to 120 sometimes.

Reply:
A related commit is this one (https://github.com/blockstack/stacks-blockchain/commit/40908325be9449afe95b93fbe50b9153f93e1164) -- I think it may fix the other neon_integrations flakiness (noted here: https://github.com/blockstack/stacks-blockchain/issues/2207). Cherry picking that commit into other branches will probably fix the gh action flake.